### ISSUE TITLE
code a PRNG on the todo list?

### ISSUE THREAD

**Issue by JeffreySarnoff** [LachlanGunn/PkgVerifierPrototype#1]
Random.jl exports MersenneTwister, would that serve your needs?

**Comment by LachlanGunn** [comment]
No, the Mersenne Twister is not cryptographically secure.  I have since written the code to access the Windows generator for another project using TweetNaCl, but haven't integrated it yet since I haven't looked into how BinDeps will cope with it on Windows.

Probably what I'll end up doing is just distributing a DLL for TweetNaCl.
